changelog/add-card-readers-inline-validation

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: add
+Comment: Add card readers business URL field browser validation
+
+
```

Comment:
The file formatting is slightly confusing, it's typically like [this](https://github.com/Automattic/woocommerce-payments/blob/develop/changelog/update-3861-ipp-exclude-l2-l3-data).

Reply:
Updated!